assets/test/specs/fixtures/form-fixture.html

```diff
@@ -1,51 +1,78 @@
-<form class="js-form" autocomplete="off" novalidate="novalidate">
-
-  <div class="flash error-summary" id="Search-failure" role="group" aria-labelledby="errorSummaryHeading" tabindex="-1">
-    <h2 id="errorSummaryHeading" class="h3-heading">There are errors on the page.</h2>
-    <ul class="js-error-summary-messages"></ul>
-  </div>
-
-  <fieldset class="form-field-group">
-    <label class="form-element-bold" for="text-input-example">Text input example
-    <span id="text-input-example-error-message" class="error-notification" role="tooltip" data-input-name="text-input-example">
-        Required text
-    </span>
-    <input type="text"
-           name="text-input-example"
-           id="text-input-example"
-           class="form-control"
-           minlength="5"
-           maxlength="13"
-           required
-           pattern="^[0-9]{5,13}$"
-           data-msg-pattern="Wrong pattern"
-           data-msg-required="Required text"
-           data-msg-minlength="Min length of 5 is required">
-    </label>
-  </fieldset>
-
-
-  <fieldset class="form-field-group" id="radio-example">
-    <span id="radio-example-error-message" class="error-notification" role="tooltip" data-input-name="radio-example">
-        Required selection
-    </span>
-
-    <label class="block-label block-label--inline" for="radio-yes">Yes
-      <input type="radio"
-             id="radio-yes"
-             value="yes"
-             name="radio-example"
+<div class="js"><!-- required to hookup css for dynamic field toggle -->
```

Comment:
Could this not be added by calling `document.body.classList.add("js")` in test setup?